transcribe/scribe_modules/lspci.py

```diff
@@ -0,0 +1,39 @@
+from . import ScribeModuleBaseClass
+
+import sys
+
+
+class Lspci(ScribeModuleBaseClass):
+
+    def __init__(self, input_dict=None, module_name=None, host_name=None,
+                 input_type=None, scribe_uuid=None):
+        ScribeModuleBaseClass.__init__(self, module_name=module_name,
+                                       input_dict=input_dict,
+                                       host_name=host_name,
+                                       input_type=input_type,
+                                       scribe_uuid=scribe_uuid)
+        if input_dict:
+            self.value = self._parse(input_dict)
+
+    def __iter__(self):
+        for attr, value in self.__dict__.items():
+            yield attr, value
+
+    def _parse(self, lspci_data):
+        output = []
+        lspci_lines = lspci_data.get("stockpile_lspci").split('\n\n')
+        if len(lspci_lines) <= 1:
+            print("Error occured in processing sysctl data")
+            sys.exit(1)
+
+        for i in range(len(lspci_lines)):
```

Comment:
Add a small detailed explanation on why you're building this dictionary this way ?